clients/benchmarks/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ target_include_directories( client
     $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
 )
 
+link_libraries( ${BOOST_ROOT}/lib )
```

Comment:
Long time ago, Paul installed cget in order to get the latest cmake on my machine